main.tf

```
@@ -1,96 +1,13 @@
-# resource "aws_cloudwatch_event_target" "sns" {
-#   rule       = "${aws_cloudwatch_event_rule.default.name}"
-#   target_id  = "SendToSNS"
-#   arn        = "${var.sns_topic_arn}"
-#   depends_on = ["aws_cloudwatch_event_rule.default"]
-#   input      = "${var.sns_message_override}"
-# }
 data "aws_caller_identity" "default" {}
 
-# Make a topic
-resource "aws_sns_topic" "default" {
-  name_prefix = "rds-threshold-alerts"
-}
-
 resource "aws_db_event_subscription" "default" {
   name_prefix = "rds-event-sub"
```

Definitely we need a name prefix so all resources with names created by this module are aligned. Additionally I think `rds-sns-alarms` is more telling.
```suggestion
  name_prefix = "${var.name_prefix}rds-sns-alarms"
```